Makefile

```diff
@@ -115,7 +115,9 @@ define PROJECT_ENV
 	    %% Default max message size is 128 MB
 	    {max_message_size, 134217728},
 	    %% Socket writer will run GC every 1 GB of outgoing data
-	    {writer_gc_threshold, 1000000000}
+	    {writer_gc_threshold, 1000000000},
+	    %% interval at which connection/channel tracking executes post operations
+	    {tracking_execution_timeout, 5000}
```

Comment:
We have seen too many times before that the 5 second timeout is too prone to false positives on a heavily loaded system.

Reply:
Currently, this timeout is being used to schedule deletion/cleanup of [tracked vhost](https://github.com/Ayanda-D/rabbitmq-server/blob/rabbitmq-per-user-connection-channel-limits/src/rabbit_connection_tracking.erl#L131) and [tracked user](https://github.com/Ayanda-D/rabbitmq-server/blob/rabbitmq-per-user-connection-channel-limits/src/rabbit_channel_tracking.erl#L113) entries (in their respective tracking tables) after `vhost_deleted` and `user_deleted` events are received, to allow & grant time for all connections and channels to first close. Idea here is that within a 5-second period terminating connections & channels for a specific user/vhost would've closed? (**note**: this is not global for all connections/channels). The false positive would be sourced from tracked connections/channels of a specific vhost/user taking longer than 5-seconds to close, which I've never come across 😅 , but happy to bump this up to a higher timeout if you think can pose an issue, or have seen such cases.